MoreLinq/EquiZip.cs

```diff
@@ -27,6 +27,8 @@ static partial class MoreEnumerable
         /// <summary>
         /// Returns a projection of tuples, where each tuple contains the N-th
         /// element from each of the argument sequences.
+        /// If the input sequences are of different lengths then
```

Comment:
Summaries generally don't reference types so let's change this to either:

```c#
        /// <summary>
        /// Returns a projection of tuples, where each tuple contains the N-th
        /// element from each of the argument sequences, if the input sequences
        /// are of different lengths.
        /// </summary>
```

If mentioning the throwing of an exception in the summary is important, then here's a version for that:

```c#
        /// <summary>
        /// Returns a projection of tuples, where each tuple contains the N-th
        /// element from each of the argument sequences. An exception is thrown
        /// if the input sequences are of different lengths.
        /// </summary>
```

I'm fine with either.